stable/xray/CHANGELOG.md

```diff
@@ -10,7 +10,7 @@ All changes to this chart will be documented in this file.
 
 ## [6.0.0] - Sep 2, 2020
 * **Breaking change:** Changed `imagePullSecrets` value from string to list.
-* **Breaking change:** Added `image.registry` and `common.xrayVersion` is changed to `image.tag` under analysis,indexer,persist,server and router sections
+* **Breaking change:** Added `image.registry` and `image.version` is changed to `image.tag` under analysis,indexer,persist,server and router sections
```

Comment:
@rahulsadanandan  please don't change previous version's changelog , Add changes in current changelog

Reply:
Its changed now.